dds_web/development/db_init.py

```diff
@@ -23,16 +23,16 @@
 # Create Units
 units = [
     Unit(
-        public_id="public_unit_id",
+        public_id="unit1",
         name="Unit 1",
-        internal_ref="fac",
-        safespring=current_app.config.get("DDS_SAFE_SPRING_PROJECT"),
+        internal_ref="someunit",
+        safespring=current_app.config.get("DDS_SAFE_SPRING_PROJECT", "dds.example.com"),
```

Comment:
Did this fix actually make a difference for you? For me, the `DDS_SAFE_SPRING_PROJECT` was set to `None` so this get call would still return `None`...

Reply:
Insted, adding a similar get call in the `config.py` file would fix the issue.

Reply:
Did some more testing and you're right, it doesn't. I have the `dds_app.cfg` file too so that's why it was filling it in.

Reply:
We already have that in the config.py. But what if the config.py file isn't used and the app config is actually done by the dds_app.cfg and DDS_SAFE_SPRING_PROJECT isn't set?

Reply:
I think `dds_app.cfg` only overwrites whatever is actually in that file. Anything else is still from the config.py, so it shouldn't be possible to have it unset then?

Reply:
Ah okey, I thought it just overwrites everything, and those that were set before are removed
But in that case, yes, I can remove this